dashboard/app/models/school_stats_by_year.rb

```diff
@@ -98,10 +98,23 @@ def has_k8_grades?
     grade_kg_offered || grade_01_offered || grade_02_offered || grade_03_offered || grade_04_offered || grade_05_offered || grade_06_offered || grade_07_offered || grade_08_offered
   end
 
+  # Percentage of underrepresented minorities students
   def urm_percent
     percent_of_students([student_am_count, student_hi_count, student_bl_count, student_hp_count].compact.reduce(:+))
   end
 
+  # Percentage of free/reduced lunch eligible students
+  def frl_eligible_percent
+    percent_of_students(frl_eligible_total)
+  end
+
+  # Is this a rural school?
+  # Returns nil if there is no data. Otherwise returns true or false.
+  def rural_school?
+    return nil unless community_type
+    community_type.starts_with? 'rural'
```

Comment:
@hacodeorg we need to cover the following community_types:`rural_fringe`, `rural_distant`, `rural_remote`, `town_remote`, `town_distant`. This looks like it would classify `town_remote` and `town_distant` as non-rural. Probably best to list these explicitly, since there is also a type that starts with "town" that we do not want to classify as rural (`town_fringe`)

Reply:
Will do. Is this logic explained somewhere or it's just universally understood?

Reply:
Should be described in the spec [here](https://docs.google.com/document/d/1CHjjb7AUUgtX0_deNoxJKeyMjD6DYHjqfQMCjKS1quQ/edit#bookmark=id.5xoxr4x1uav3) and also as a footnote in [our rubric ](https://drive.google.com/file/d/1070Jf9VKtuJLOQJTCaO7fxUWyLOEHBdK/view). It sounds like we're basing this off of a federal program (REAP) definition

Reply:
Fixed!